config/env.js

```diff
@@ -1,6 +1,8 @@
 export const HOST = process.env.HOSTNAME || 'localhost';
 export const PORT = process.env.PORT || '3000';
 export const ENV = process.env.NODE_ENV || 'development';
+export const BACKEND_URL = process.env.BACKEND_URL ||
+  (ENV === 'development' ? 'http://localhost:3000/graphql' : 'https://life-balance.herokuapp.com/graphql');
```

Comment:
I'd suggest to reuse `HOST` and `PORT` variables here instead of duplicating theirs content.